app/routes/help.js

```diff
@@ -0,0 +1,4 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+});
```

Comment:
I think this could be deleted 😀

Reply:
This is auto-generated by Ember CLI. Wouldn't it make sense to keep such things, in order to make sure that everything works as it was designed? For instance, you won't be able to inject a controller unless you have it explicitly generated (even empty). I can't imagine any problems with routes at the moment, but in sake of stability and upgradability I usually strive to stick with framework defaults. Does it make sense in your opinion?

Reply:
oh yes, I almost never use generators so I forget about these things haha. It’s not a huge deal to keep this.